sos/plugins/ovirt_hosted_engine.py

```diff
@@ -48,6 +48,14 @@ def setup(self):
             '/var/log/ovirt-hosted-engine-ha/agent.log',
             '/var/log/ovirt-hosted-engine-ha/broker.log',
         ])
+
+        # Add gluster deployment and cleanup log
+        self.add_copy_spec([
+            '/var/log/cockpit/ovirt-dashboard',
```

Comment:
Collecting whole dir implies collecting the two particular logfiles from the dir - didnt you want to collect some different directory here?

Reply:
No, The logfiles which are mentioned under '/var/log/cockpit/ovirt-dashboard' need to be included in sosreport.

Reply:
Right, so you only need the first line collecting the dir. Everything in that directory will be collected, including those files.

Reply:
@pmoravec Ok got it. I have pushed the changes.